packages/wallet/src/redux/protocols/indirect-funding/player-a/state.ts

```diff
@@ -16,6 +17,10 @@ interface LedgerChannelExists extends BasePlayerAState {
   ledgerId: string;
 }
 
+interface DirectFundingExists extends LedgerChannelExists {
```

Comment:
I don't think this interface is needed, since it will only ever be used once. Therefore, `WaitForDirectFunding` can extend `LedgerChannelExists`, and simply add a `directFundingState` property itself.